image-loader/app/lib/FailureResponse.scala

```diff
@@ -54,6 +54,16 @@ object FailureResponse extends ArgoHelpers with Results {
     )
   }
 
+  def imageAlreadyExists(exception: Exception): Response = {
+    logger.info(s"Rejected request to load file: image file is already exists and soft deleted", exception)
+
+    Response(
+      BadRequest,
+      "image-already-exists",
+      s"Image already exists: image file already exists and soft deleted."
```

Comment:
Same here

Reply:
Same as above? `state is good for logs and API responses, but can be dropped in user-facing warnings, I guess?